entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -2,18 +2,35 @@
 title: Migrations - EF Core
 author: bricelam
 ms.author: bricelam
-ms.date: 10/30/2017
+ms.date: 10/05/2018
 uid: core/managing-schemas/migrations/index
 ---
 Migrations
 ==========
-Migrations provide a way to incrementally apply schema changes to the database to keep it in sync with your EF Core
-model while preserving existing data in the database.
 
-Creating the database
----------------------
-After you've [defined your initial model][1], it's time to create the database. To do this, add an initial migration.
-Install the [EF Core Tools][2] and run the appropriate command.
+A data model changes during development and gets out of sync with the database. You can drop the database and let EF create a new one that matches the model, but this procedure results in the loss of data. The EF Core Migrations feature preserves data by incrementally updating the database schema instead of creating a new database.
```

Comment:
Nit: Can we replace the last sentence with:

> The migrations feature in EF Core provides a way to incrementally update the database schema to keep it in sync with your application's data model while preserving existing data in the database.

It is a mix with the sentence that was there before. I think it conveys the purpose of the feature a bit more clearly.